kodilla-patterns2/src/main/java/com/kodilla/patterns2/adapter/bookclasifier/MedianAdaptee.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.patterns2.adapter.bookclasifier;
+
+import com.kodilla.patterns2.adapter.bookclasifier.libraryb.Book;
+import com.kodilla.patterns2.adapter.bookclasifier.libraryb.BookSignature;
+import com.kodilla.patterns2.adapter.bookclasifier.libraryb.BookStatistics;
+import com.kodilla.patterns2.adapter.bookclasifier.libraryb.Statistics;
+
+import java.util.Map;
+
+public class MedianAdaptee implements BookStatistics {
+
+    @Override
+    public int averagePublicationYear(Map<BookSignature, Book> books) {
+        Statistics statistics = new Statistics();
```

Comment:
to może być pole klasy - dwa razy tworzysz taki sam obiekt Statistics w każdej z metod.